EnRonSwanson/BangazonAPI
# Mitchell

Fixes #8 and #9 .

Changes proposed in this pull request:

- Added Get (all and single), Post, Put, Delete functionality to Computer Controller
- Added Get (all and single), Post, Put functionality to TrainingProgram Controller

How to test:
Run all calls in postman or terminal and confirm changes are identical in browser / db

@EnRonSwanson/
